### PR TITLE
[KTL-124] feature new header Kolinlang + WebHelp

### PR DESCRIPTION
**Pull request by JuliaVolkova**

### CODE REVIEW

**Review by dmitry-konchalenkov — APPROVED** [review]
LGTM!

File: static/css/com/layout/header.scss [hunk 1]
```diff
@@ -9,6 +9,15 @@
   background-color: #27282c;
   background-size: cover;
   min-width: 960px;
+  position: relative;
+  display: flex;
```
Comment (zoobestik) [inline static/css/com/layout/header.scss:13]: I'm not sure, why are we use `flex` for `global-header`? Cause, it's has just one child with full size.

File: static/css/com/layout/header.scss [hunk 2]
```diff
@@ -9,6 +9,15 @@
...
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 64px;
```
Comment (zoobestik) [inline static/css/com/layout/header.scss:18]: 🤔  do we need it?

File: static/css/com/layout/header.scss [hunk 3]
```diff
@@ -94,29 +104,31 @@
 }
 
 .global-header-panel {
-  @include clearfix;
   box-sizing: border-box;
-  padding: $unit 0;
-  height: 60px;
+  display: flex;
+  width: 100%;
```
Comment (zoobestik) [inline static/css/com/layout/header.scss:109]: If `global-header` does not have flex, it won't required, yes?

File: static/css/com/layout/header.scss [hunk 4]
```diff
@@ -94,29 +104,31 @@
...
+  padding: 0 22px;
+  align-items: center;
+  justify-content: space-between;
+  height: 64px;
```
Comment (zoobestik) [inline static/css/com/layout/header.scss:113]: Same question?

File: static/css/com/nav/global-nav.scss [hunk 5]
```diff
@@ -9,35 +9,57 @@
   position: relative;
   float: right;
```
Comment (zoobestik) [inline static/css/com/nav/global-nav.scss:10]: May we clean old styles?

File: static/css/com/nav/global-nav.scss [hunk 6]
```diff
@@ -9,35 +9,57 @@
...
   font-size: 0;
-  margin-top: 2px;
+  height: 100%;
```
Comment (zoobestik) [inline static/css/com/nav/global-nav.scss:12]: What is problem it resolve?

File: templates/inc/nav/global-nav.html [hunk 7]
```diff
@@ -1,12 +1,54 @@
...
+                                                       href="https://www.jetbrains.com/lp/mobilecrossplatform/"
+                                                       target="_blank">Mobile cross-platform ↗</a></li>
+                    <li class="solutions-menu-item">
+                        <a class="solutions-menu-item-link" href="https://kotlinlang.org/lp/server-side/"
+                           target="_blank">
+                            Server-side ↗
+                        </a></li>
+                    <li class="solutions-menu-item">
+                        <a class="solutions-menu-item-link"
+                           href="https://kotlinlang.org/docs/reference/native-overview.html"
```
Comment (zoobestik) [inline templates/inc/nav/global-nav.html:17]: Drop domain, pls

File: static/css/com/nav/global-nav.scss [hunk 8]
```diff
@@ -180,8 +207,74 @@
...
     }
   }
+}
+
+.solutions-menu {
+  width: 260px;
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  background-color: #27282C;
+  position: absolute;
+  top: 70px;
```
Comment (zoobestik) [inline static/css/com/nav/global-nav.scss:222]: It's looks like hard dependency on menu size, may be it's better to use `bottom`?

File: static/css/com/layout/header.scss (hunk 1 above)
Comment (JuliaVolkova) [reply static/css/com/layout/header.scss:13]: It's redundant, I've removed it, thanks)

File: static/css/com/layout/header.scss (hunk 2 above)
Comment (JuliaVolkova) [reply static/css/com/layout/header.scss:18]: It's too, removed it.

File: static/css/com/layout/header.scss (hunk 3 above)
Comment (JuliaVolkova) [reply static/css/com/layout/header.scss:109]: Yes, removed it too)

File: static/css/com/nav/global-nav.scss (hunk 5 above)
Comment (JuliaVolkova) [reply static/css/com/nav/global-nav.scss:10]: Fixed.

File: static/css/com/nav/global-nav.scss (hunk 6 above)
Comment (JuliaVolkova) [reply static/css/com/nav/global-nav.scss:12]: Fixed it, now use  `align-items: stretch`.

File: templates/inc/nav/global-nav.html (hunk 7 above)
Comment (JuliaVolkova) [reply templates/inc/nav/global-nav.html:17]: Fixed.

File: static/css/com/nav/global-nav.scss (hunk 8 above)
Comment (JuliaVolkova) [reply static/css/com/nav/global-nav.scss:222]: It makes sense, fixed it, now use `bottom`.

File: static/css/com/layout/header.scss (hunk 4 above)
Comment (JuliaVolkova) [reply static/css/com/layout/header.scss:113]: We need it to set header height.
And we can't use min-height cause we have banner in this header :(

**Review by zoobestik — APPROVED** [review]

**Review by zoobestik — APPROVED** [review]

File: static/css/com/nav/global-nav.scss (hunk 8 above)
Comment (zoobestik) [reply static/css/com/nav/global-nav.scss]: use variables in this place and header height looks better, but it's ok too

**Pull request merged**